res/DegenEmbed.py

```diff
@@ -0,0 +1,25 @@
+import discord
+
+
+class DegenEmbed:
```

Comment:
why don't you just extend Embed?

Reply:
I'm not as familiar with extending classes in Discord. Not sure the benefit it offers. I can always refactor it a little later.